[docs] Standardize getting started docs across all packages (#16302)
mui/mui-x: docs/data/date-pickers/quickstart/quickstart.md

---
productId: x-date-pickers
title: Date and Time Pickers - Quickstart
packageName: '@mui/x-date-pickers'
githubLabel: 'component: pickers'
materialDesign: https://m2.material.io/components/date-pickers
waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/
---

# Date and Time Pickers - Quickstart

<p class="description">Install the MUI X Date and Time Pickers package and set up your date library to start building.</p>

## Installation

> Install the Date Pickers package that best suits your needs—Community or Pro—along with a required third-party date library.

Reviewer: As a non native english speaker, I fearthat "along with a required" is not explicit enough.

And for the name of the package, if we can keep "and Time" to be consistent with the other occurences I think it would be better

```suggestion
Install the Date and Time Pickers package that best suits your needs—Community or Pro—along with one of the supported third-party date library.
```

Author: Good call—it didn't occur to me that that phrasing would be unclear to a non-native speaker, so thanks for pointing that out.

```suggestion
Install the Date and Time Pickers package that best suits your needs—Community or Pro—along with one of the supported third-party date libraries.
```

import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
import { DatePicker } from '@mui/x-date-pickers/DatePicker';

export default function App() {
  return (
    <LocalizationProvider dateAdapter={AdapterDayjs}>
      <DatePicker />
    </LocalizationProvider>
  );
}
```

{{"demo": "FirstComponent.js"}}

## TypeScript

### Theme augmentation

To benefit from [CSS overrides](/material-ui/customization/theme-components/#theme-style-overrides) and [default prop customization](/material-ui/customization/theme-components/#theme-default-props) with the theme, TypeScript users must import the following types.
These types use module augmentation to extend the default theme structure.

```tsx
// Pro users: add `-pro` suffix to package name
import type {} from '@mui/x-date-pickers/themeAugmentation';

const theme = createTheme({
  components: {
    MuiDatePicker: {
      defaultProps: {
        displayWeekNumber: true,
      },
    },
    MuiDateRangeCalendar: {
      styleOverrides: {
        root: {
          backgroundColor: '#f0f0f0',
        },
      },
    },
  },
});
```

> ### Date and time types

Author: ```suggestion
### Date value types
```
